### ISSUE TITLE
Craft 4 - Google Signin auto login

### ISSUE THREAD

**Issue by franck-thorck** [verbb/social-login#15]
### Describe the bug

Hi,

I have an issue, when I used the Google Signin on the CP login page, it works by creating the new user but it redirect me to the CP login page and not on the admin panel directly.
When I try to overwrite some configuration with social-login.php in my config folder, it does not work too. The settings are not overwritten.

The connect button not works in the user account in the admin panel too...

If someone can help me about that.

### Steps to reproduce

1. Configure Client ID and Client Secret on Google Developer Console
2. Configure Social Login settings and Google provider
3. Login from CP Login

### Craft CMS version

4.6.0

### Plugin version

3.0.0-beta.1

### Multi-site?

Yes

**Comment by engram-design** [comment]
Just wanted to confirm your plugin version, as there is no 3.0.0-beta.1?

In [1.0.9](https://github.com/verbb/social-login/releases/tag/1.0.9) we only just recently added registration for control panel users, which you'll want to be careful with, as essentially anyone with a Google account can have a user account created on your site. They can't automatically access your control panel without proper permissions though.

As for the config file not applying, what settings are you trying to apply? What does the file look like? There should be an orange notice on any setting that's being overridden via config file, when visiting Settings > Social Login.

**Comment by franck-thorck** [comment]
Yes it is 3.0.0-beta.1 displayed on the plugins section in the control panel and in my composer.json:

`"dukt/social": "3.0.0-beta.1"`

I installed it from the plugin store from the control panel.
I also tried to setup the default user group in the plugin settings, but when I used the Google Signin the user created had not the user group assigned.

For the config file I copied the default one from the plugin documentation and change just some settings to try, there was no orange notice on settings in the control panel for Social Login settings. When I saw the configuration file was not working, I just deleted it.
